### DISCUSSION TITLE
Could someone help me generate a steroscopic video

### DISCUSSION THREAD

**Discussion by aplayz522** [thygate/stable-diffusion-webui-depthmap-script#466] [Q&A]
Hi everyone,

I've got a .mp4 file and i want to generate it into a stereoscopic video. how do i do this? i cant find one simple tutorial on this. could someone tell me the step-by-step process?

Thanks in advance :)

**Comment by donbrasco01** [comment]
1. Click"Single Video" Tab
2. Drag&Drop video into "Video or animated file"
3. Click "Compress colorvideos?"
4. Choose "Model"
5. Click "Generate stereoscopic (2D) image(s)"
5.1. Choose parameters

Click the big orange button
